Update packages & setup docker-compose to allow override.env file (#239)
navapbc/template-application-flask: docs/app/README.md

Most configuration options are managed by environment variables.

Environment variables for local development are stored in the [local.env](/app/local.env) file. This file is automatically loaded when running. If running within Docker, this file is specified as an `env_file` in the [docker-compose](/app/docker-compose.yml) file, and loaded [by a script](/app/src/util/local.py) automatically when running unit tests (see running natively above for other cases).

> To override environment variables for locally, create an `override.env` file in the same directory as local.env. The [docker-compose file](https://docs.docker.com/reference/compose-file/services/#required) optionally checks this file as well.

Reviewer: ```suggestion
To override environment variables for locally, create an `override.env` file in the same directory as local.env. The docker-compose file [optionally checks this file](https://docs.docker.com/reference/compose-file/services/#required) as well.
```

a nit to move the hyperlink to the part of the text that it explains

Reviewer: (this might not actually be clearer so feel free to ignore)

> If you would like to override environment variables for your own local development, create an `override.env` file in the same directory as local.env. The [docker-compose file](https://docs.docker.com/reference/compose-file/services/#required) optionally checks this file as well.

Reviewer 2: nit
```suggestion
To override environment variables locally, create an `override.env` file in the same directory as local.env. The [docker-compose file](https://docs.docker.com/reference/compose-file/services/#required) optionally checks this file as well.
```
